static/app/views/alerts/wizard/panelContent.tsx

```diff
@@ -147,4 +147,64 @@ export const AlertWizardPanelContent: Record<AlertType, PanelContent> = {
     ],
     illustration: diagramCrashFreeUsers,
   },
+  app_start_cold: {
+    description: t('App cold start metric track when your app takes too long to launch.'),
+    examples: [
+      t(
+        'On iOS, Apple recommends your app take at most 400ms to render the first frame.'
+      ),
+      t(
+        'On Android, the Google Play console warns you when a cold start takes longer than five seconds or a warm start longer than two seconds.'
+      ),
+    ],
+    illustration: diagramCustom,
+  },
+  app_start_warm: {
+    description: t(
+      'App warm start metric track when your app takes does not too long to launch.'
+    ),
+    examples: [
+      t(
+        'On iOS, Apple recommends your app take at most 400ms to render the first frame.'
+      ),
+      t(
+        'On Android, the Google Play console warns you when a cold start takes longer than five seconds or a warm start longer than two seconds.'
+      ),
+    ],
+    illustration: diagramCustom,
+  },
+  frames_slow: {
+    description: t(
+      'Using 60 fps, slow frames are frames that take more time than necessary to render.'
+    ),
+    examples: [
+      t('On iOS, a slow frame occurs when a frame renders after 16.67 ms.'),
+      t('On Android, a slow frame occurs when a frame renders after 16 ms.'),
+    ],
+    illustration: diagramCustom,
+  },
+  frames_frozen: {
+    description: t('Frozen frames are frames that take longer than 700 ms to render.'),
+    examples: [
+      t(
+        'For Apple, the frame rate can be higher, especially as 120 fps displays are becoming more popular. For these apps, Sentry detects the frame rate and adjusts the slow frame calculation accordingly.'
```

Comment:
This description should be in `frames_slow`. For frozen frames, the frame rate has no impact.